overview/requirements.md

```diff
@@ -72,31 +71,28 @@ needs to be.
     - Encoders sometimes deal with files up to 40GB each and
       regularly with ones between 1GB and 8GB and may be required
       to retain these files for a long time.
-  - **Internet**: 25 Mbps download, 10 Mbps upload (faster
-    recommended)
-- **Editor**
-    - At least the same as the Timer role.
-- **Quality Checker**
-    - At least the same as the Typesetter role.
-- **Translator / Translation Checker**
-    - At least the same as the Timer role.
+  - **Internet**: 25 Mbps download, 25 Mbps upload
 
 ### Software
 
 Every role will have different required software, but it is recommended
-for every role to have installed [Aegisub](http://www.aegisub.org). It
-is highly recommended to use line0's builds, which can be found
-[here](https://files.line0.eu/builds/Aegisub/). They include a
-preinstalled automation script package manager and several critical
-fixes to aegisub that have not been merged into the official
+for every role to have installed [Aegisub][]. It
+is **highly** recommended to use [line0's builds][]. They include a
```

Comment:
We should change this once Ryan/Coffee got a stable build out, but for now, line0 has mirrored Ryan's latest. Just a note for the future.